config/config.yml

```diff
@@ -32,64 +32,25 @@ en:
     - "B-MISC"
     - "I-MISC"
     - "O"
-  model:           'bert-base-cased'
+  model:             'bert-base-cased'
 
 ru:
-  train_data_path: "../data/ru/combined/devset_combined.txt"
-  dev_data_path:   "../data/ru/combined/testset_combined.txt"
+  train_data_path:   "../data/ru/combined/devset_combined.txt"
+  dev_data_path:     "../data/ru/combined/testset_combined.txt"
   label_types:
-      - "PER"
-      - "LOC"
-      - "ORG"
-      - "MISC"
-      - "EVT"
-      - "PRO"
-      - "O"
-    # - "B-geo_adj"
-    # - "B-job"
-    # - "B-loc_descr"
-    # - "B-loc_name"
-    # - "B-name"
-    # - "B-nickname"
-    # - "B-org_descr"
-    # - "B-org_name"
-    # - "B-patronymic"
-    # - "B-prj_name"
-    # - "B-surname"
-    # - "I-geo_adj"
-    # - "I-job"
-    # - "I-loc_descr"
-    # - "I-loc_name"
-    # - "I-name"
-    # - "I-nickname"
-    # - "I-org_descr"
-    # - "I-org_name"
-    # - "I-surname"
-    # - "L-geo_adj"
-    # - "L-job"
-    # - "L-loc_descr"
-    # - "L-loc_name"
-    # - "L-name"
-    # - "L-nickname"
-    # - "L-org_descr"
-    # - "L-org_name"
-    # - "L-patronymic"
-    # - "L-prj_name"
-    # - "L-surname"
-    # - "U-geo_adj"
-    # - "U-job"
-    # - "U-loc_descr"
-    # - "U-loc_name"
-    # - "U-name"
-    # - "U-nickname"
-    # - "U-org_descr"
-    # - "U-org_name"
-    # - "U-patronymic"
-    # - "U-prj_descr"
-    # - "U-surname"
-    # - "O"
-  model:           'bert-base-multilingual-cased'
+    - "PER"
+    - "LOC"
+    - "ORG"
+    - "MISC"
+    - "EVT"
+    - "PRO"
+    - "O"
+  model:             'bert-base-multilingual-cased'
 
 demo_text:
-  en:              On this day in 1957, a California Superior Court judge ruled that "Howl", a poem by Allen Ginsberg, was of "redeeming social importance" and thus not obscene.
-  ru:              МОСКВА, 4 авг - РИА Новости. Россия рассчитывает, что США воздействуют на Тбилиси в связи с обострением ситуации в зоне грузино-осетинского конфликта. Об этом статс-секретарь - заместитель министра иностранных дел России Григорий Карасин заявил в телефонном разговоре с заместителем госсекретаря США Дэниэлом Фридом.
+  en:                On this day in 1957, a California Superior Court judge ruled that "Howl", a poem by Allen Ginsberg, was of "redeeming social importance" and thus not obscene.
+  ru:                МОСКВА, 4 авг - РИА Новости. Россия рассчитывает, что США воздействуют на Тбилиси в связи с обострением ситуации в зоне грузино-осетинского конфликта. Об этом статс-секретарь - заместитель министра иностранных дел России Григорий Карасин заявил в телефонном разговоре с заместителем госсекретаря США Дэниэлом Фридом.
+
+parallel_demo_text:
+  en:                Lately, with gold prices up more than 300% over the last decade, it is harder than ever. Just last December, fellow economists Martin Feldstein and Nouriel Roubini each penned op-eds bravely questioning bullish market sentiment, sensibly pointing out gold’s risks.
```

Comment:
Whoops, thanks for pointing this out! The data source claims that the parallel text is aligned by line number, but I guess an exact match wasn't possible here (it looks like the English has it split over two sentences, while in the Russian version this sentence is part of the previous sentence). I'll just remove that sentence from the English example since we're not losing any entities.